src/main/java/teammates/ui/template/InstructorFeedbackResultsQuestionTable.java

```diff
@@ -37,12 +34,13 @@
     private List<ElementTag> columns;
     private Map<String, Boolean> isColumnSortable;
 
-    public InstructorFeedbackResultsQuestionTable(InstructorFeedbackResultsPageData data,
-                                          List<FeedbackResponseAttributes> responses,
+    public InstructorFeedbackResultsQuestionTable(
+                                          boolean isQuestionHasResponses,
```

Comment:
The "Question" part of its name can probably be eliminated since this is already the **Question**Table.